### ISSUE TITLE
[Dropdown]: activedescendent not de-selected on blur

### ISSUE THREAD

**Issue by scottnath** [carbon-design-system/carbon#4031]
## Environment

> Browser
- all
> Automated testing tool and ruleset
- DAP + IBMa/aat

## Detailed description

> What version of the Carbon Design System are you using?
10.6x

> What did you expect to happen?
When clicking away from a dropdown (blur), the `aria-activedescendent` attribute should be removed from the menu (ul.bx--dropdown).

This behavior is there if you `close` the dropdown by clicking it, it just needs to be added on `blur` of the dropdown - which _also_ closes the dropdown, but _does_ remove the attribute. See [this section in the dropdown vanilla js](
https://github.com/carbon-design-system/carbon/blob/master/packages/components/src/components/dropdown/dropdown.js#L184) where the attribute is removed

> What happened instead?

The attribute is not removed on `blur`, causing the DAP/AAT violation

https://www.w3.org/TR/wai-aria-1.1/#aria-activedescendant

## Acceptance Criteria

- [ ] `aria-activedescendent` is removed on dropdown blur

**Comment by elizabethsjudd** [comment]
@dakahn I can work on this issue if you want to assign it to me

**Comment by elizabethsjudd** [comment]
@carbon-design-system/developers I have some free time that I could contribute this fix if it's assigned to me. It's blocking our release.
